lib/board.ex

```diff
@@ -27,4 +27,7 @@ defmodule Board do
     Enum.filter(board, fn(space) -> space != "X" and space != "O" end)
   end
 
+  def reset_space(board, space) do
+    List.replace_at(board, space, space)
+  end
```

Comment:
Just out of curiosity--what is the purpose of this function? It looks like it's replacing one space with the exact same one.